smartsim/_core/launcher/slurm/slurmLauncher.py

```diff
@@ -149,17 +149,22 @@ def run(self, step: Step) -> t.Optional[str]:
                 logger.debug(f"Gleaned batch job id: {step_id} for {step.name}")
 
         # Launch a in-allocation or on-allocation (if srun) command
+        elif isinstance(step, SrunStep):
+            task_id = self.task_manager.start_task(cmd_list, step.cwd)
         else:
-            if isinstance(step, SrunStep):
-                task_id = self.task_manager.start_task(cmd_list, step.cwd)
-            else:
-                # Mpirun doesn't direct output for us like srun does
-                out, err = step.get_output_files()
-                output = open(out, "w+", encoding="utf-8")  # pylint: disable=consider-using-with
-                error = open(err, "w+", encoding="utf-8")  # pylint: disable=consider-using-with
-                task_id = self.task_manager.start_task(
-                    cmd_list, step.cwd, out=output.fileno(), err=error.fileno()
-                )
+            # MPI/local steps don't direct output like slurm steps
+            out, err = step.get_output_files()
+
+            # LocalStep.run_command omits env, include it here
```

Comment:
Suggest leaving a comment here that `None` means that the child process inherits the caller's environment by default

Reply:
Love the idea, but I instead updated the doc string for `TaskManager.start_task` so the env parameter states this behavior. I think this could be more helpful than a comment:

![image](https://github.com/CrayLabs/SmartSim/assets/3595025/43ea1813-5297-4f01-9fdc-41982d2b141f)